apps/zbugs/shared/queries.ts

```diff
@@ -55,15 +73,23 @@ export const queries = {
     builder.user.where('id', userID).one(),
   ),
 
+  crewUser: defineQuery(idValidator, ({args: userID}) =>
```

Comment:
I guess, but I actually liked it just getting the current user and checking the role. It doesn't seem to add anything to have a query? And remember we're trying to minimize number of queries.